sdk/hdinsight/Azure.ResourceManager.HDInsight/api/Azure.ResourceManager.HDInsight.netstandard2.0.cs

```diff
@@ -936,7 +936,7 @@ public partial class HDInsightPrivateLinkServiceConnectionState
     public partial class HDInsightSecurityProfile
     {
         public HDInsightSecurityProfile() { }
-        public string AaddsResourceId { get { throw null; } set { } }
+        public Azure.Core.ResourceIdentifier AaddsResourceId { get { throw null; } set { } }
```

Comment:
Just out of curiosity, which resource type is this ID?

Reply:
I guess it's `Microsoft.AzureActiveDirectory*`